examples/ammonia-cat.rs

```diff
@@ -29,7 +29,7 @@ fn run() -> io::Result<()> {
 fn main() {
     env_logger::init();
     if let Err(ref e) = run() {
-        println!("error: {}", e);
+        eprintln!("error: {}", e);
```

Comment:
I guess we might as well:

```suggestion
        eprintln!("error: {e}");
```

I think it's available since 1.58, our MSRV is 1.60.